test/e2e/e2e.go

```diff
@@ -32,11 +32,16 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/v1"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
```

Comment:
Is this what we're doing now?  Seems weird that we're hard coding version into clientset? 

/cc @deads2k